fmriprep/cli/run.py

```diff
@@ -20,75 +17,94 @@
 from multiprocessing import cpu_count
 from time import strftime
 
-def main():
-    """Entry point"""
-    from fmriprep import __version__
-    parser = ArgumentParser(description='fMRI Preprocessing workflow',
+def get_parser():
+    """Build parser object"""
+
+    from fmriprep.info import __version__
+    verstr = 'fmriprep v{}'.format(__version__)
+
+    parser = ArgumentParser(description='FMRIPREP: fMRI PREProcessing workflows',
                             formatter_class=RawTextHelpFormatter)
 
     # Arguments as specified by BIDS-Apps
     # required, positional arguments
     # IMPORTANT: they must go directly with the parser object
-    parser.add_argument('bids_dir', action='store', default=os.getcwd())
+    parser.add_argument('bids_dir', action='store',
+                        help='the root folder of a BIDS valid dataset (sub-XXXXX folders should '
+                             'be found at the top level in this folder).')
     parser.add_argument('output_dir', action='store',
-                        default=op.join(os.getcwd(), 'out'))
-    parser.add_argument('analysis_level', choices=['participant'])
+                        help='the output path for the outcomes of preprocessing and visual '
+                             'reports')
+    parser.add_argument('analysis_level', choices=['participant'],
+                        help='processing stage to be run, only "participant" in the case of '
+                             'FMRPREP (see BIDS-Apps specification).')
 
     # optional arguments
-    parser.add_argument('--participant_label', action='store', nargs='+')
-    parser.add_argument('-v', '--version', action='version',
-                        version='fmriprep v{}'.format(__version__))
-
-    # Other options
-    g_input = parser.add_argument_group('fMRIprep specific arguments')
-    g_input.add_argument('-s', '--session-id', action='store', default='single_session')
-    g_input.add_argument('-r', '--run-id', action='store', default='single_run')
-    g_input.add_argument('--task-id', help='limit the analysis only ot one task', action='store')
-    g_input.add_argument('-d', '--data-type', action='store', choices=['anat', 'func'])
-    g_input.add_argument('--debug', action='store_true', default=False,
+    parser.add_argument('-v', '--version', action='version', version=verstr)
+
+    g_bids = parser.add_argument_group('Options for filtering BIDS queries')
+    g_bids.add_argument('--participant_label', action='store', nargs='+',
+                        help='one or more participant identifiers (the sub- prefix can be '
```

Comment:
s/can/should/

I don't think we remove `sub-` and some things assume it's not there. (Unless you've added sanitization and I haven't gotten there yet.)